rust/template/differential_datalog/program.rs

```diff
@@ -1618,9 +1627,10 @@ impl Program {
 
     /* Profiler thread function */
     fn prof_thread_func(chan: mpsc::Receiver<ProfMsg>, profile: Arc<Mutex<Profile>>) {
+        let mut stats = Statistics::new("stats.csv").expect("TODO");
```

Comment:
This thread seems to be spawned whether or not profile_cpu is on? And it merely blocks on `recv()` forever if profiling isn't on. This means that the `stat.csv` file is always created but without cpu profiling it is empty. Is this okay? I can change the logic so the empty file isn't created when logging is off.